plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -3070,14 +3068,16 @@ public void testEmptyTableCreation()
     public void testCreateEmptyTableShouldNotCreateStagingDirectory()
             throws IOException
     {
+        HiveConfig config = getHiveConfig();
+        String tableOwner = "random-user-" + randomNameSuffix();
+        ConnectorSession session = newSession(ImmutableMap.of("user", tableOwner));
+
         for (HiveStorageFormat storageFormat : createTableFormats) {
             SchemaTableName temporaryCreateEmptyTable = temporaryTable("create_empty");
             try {
                 List<Column> columns = ImmutableList.of(new Column("test", HIVE_STRING, Optional.empty()));
                 try (Transaction transaction = newTransaction()) {
-                    final String temporaryStagingPrefix = "hive-temporary-staging-prefix-" + UUID.randomUUID().toString().toLowerCase(ENGLISH).replace("-", "");
-                    ConnectorSession session = newSession(ImmutableMap.of("hive.temporary_staging_directory_path", temporaryStagingPrefix));
-                    String tableOwner = session.getUser();
+                    final String temporaryStagingPrefix = config.getTemporaryStagingDirectoryPath().replace("${USER}", tableOwner);
```

Comment:
can this create a new temporaryStagingPrefix path that is valid and could be `mkdir`, but doesn't exist?
```
java.nio.file.Path uniqueStagingDirectory = createTempDirectory("new-unique-location");
Files.delete(uniqueStagingDirectory);
```

without randomizing `tableOwner` and without `getHiveConfig` + `getTemporaryStagingDirectoryPath` + `replace`?